x-pack/plugins/alerts/server/usage/alerts_usage_collector.ts

```diff
@@ -50,6 +50,29 @@ export function createAlertsUsageCollector(
         };
       }
     },
+    schema: {
+      count_total: { type: 'long' },
+      count_active_total: { type: 'long' },
+      count_disabled_total: { type: 'long' },
+      throttle_time: {
+        min: { type: 'long' },
+        avg: { type: 'float' },
+        max: { type: 'long' },
+      },
+      schedule_time: {
+        min: { type: 'long' },
+        avg: { type: 'float' },
+        max: { type: 'long' },
+      },
+      connectors_per_alert: {
+        min: { type: 'long' },
+        avg: { type: 'float' },
+        max: { type: 'long' },
+      },
+      // TODO: Find out all the possible values for DYNAMIC_KEY or reformat into an array
```

Comment:
Cross referencing a discussion from https://github.com/elastic/kibana/pull/78832#discussion_r497520714 in regards to if these can be gathered somehow. This one plugins do call `registerType` for their own types of alerts.

Reply:
Thank you @mikecote! I'll run through the code to find all the known registered types.

Do you think we can still keep the extra `DYNAMIC_KEY` entry to let the Remote Telemetry team know that it might grow in the future?